papaya-web/src/main/java/pl/edu/agh/papaya/notification/NotificationService.java

```diff
@@ -28,18 +28,13 @@
 @SuppressWarnings({"PMD.BeanMembersShouldSerialize"})
 public final class NotificationService {
 
-    @Value("${notifications.frequency.minutes:1440}")
```

Comment:
U @kjarosh też się nagle przeformatowały te pola. U mnie tak nie robi :(
Chyba raczej zostawiamy oddzielone newline'ami

Reply:
Możesz zostawić pola w tej kolejności i oddzielić tylko newline'ami? Formater powinien to zaakceptować, a wszędzie raczej mamy rozdzielone